browser/brave_stats_updater.cc

```diff
@@ -45,7 +45,7 @@ std::string GetPlatformIdentifier() {
   std::string os_type = version_info::GetOSType();
   if (os_type == "Linux")
     return kPlatformIdentifierLinux;
-  else if (os_type == "Mac")
+  else if (os_type == "Mac OS X")
```

Comment:
@bbondy Relying on string comparisons here seems brittle. Is there a better approach I should take? I wasn't able to find any helpful function, short of just reimplementing `version_info::GetOSType` here.

Reply:
what about compile time defines?

Reply:
like  `#if defined(OS_LINUX)`

Reply:
Yep, that makes sense.